rgw/v2/tests/s3_swift/test_quota_management.py

```diff
@@ -0,0 +1,210 @@
+"""test_quota_management - Test quota management
+
+Usage: test_quota_management.py -c <input_yaml>
+
+<input_yaml>
+        Note: any one of these yamls can be used
+        test_quota_management.yaml
```

Comment:
Please add all supported yaml list here for future referance

Reply:
done uday